website/landing/website.config.json

```diff
@@ -56,19 +56,19 @@
       {
         "name": "Twitter",
         "socialUrl": "https://twitter.com/codesandbox",
-        "description": "Lorem Ipsum is simply dummy of the printing and typesetting industry. Lorem Ipsum has.",
+        "description": "Sandpack wouldn't exist without this fantastic community of developers. Follow @codesandbox on Twitter to get the latest updates.",
```

Comment:
Do you think there should be some sort of highlight on the `@sandpack`? cc @marcovincit